### ISSUE TITLE
Can't get Inky to work on  Raspberry Pi Zero 2 W

### ISSUE THREAD

**Issue by BoardGames482** [pimoroni/inky#186]
_**I will update/edit this with more information later this week when I can copy the exact output from my Pi**_

I am trying to get Inky to work with a 5.7" (600 x 448 pixel) Inky Impression screen.
I just flashed a new SD card with bullseye (v11) for my [Raspberry Pi Zero 2 W](https://www.raspberrypi.com/products/raspberry-pi-zero-2-w/)
I'm running into some confusing issues and hope you can help.

Depending on the example I run (clone repo -> cd into examples dir -> `python3 clear.py` , for example) I am stopped by an import issue due to SMBus or the Hanken fonts.

- Failure to import `SMBus from smbus2`

```
< will fill in with more detail later this week > 
```
-  Failure to import `HankenGroteskBold from font_hanken_grotesk`

```
< will fill in with more detail later this week > 
```

Things I have tried:

- using the bash script: `curl https://get.pimoroni.com/inky | bash` (it seemed to detect ISC and SPI and all the packages I had installed with no issues)
- installing with the `rpi` and `example-depends` options: `pip3 install inky[rpi,example-depends]`
- enabling SPI and ISC through the `rpi-config` 
- installing packages through `apt` and `pip`
- some other steps that I forgot (will try to update later)

**Comment by JeppeJonHolt** [comment]
I have the same problem

**Comment by BoardGames482** [comment]
@JeppeJonHolt thank you very much this did solve my issue.

For any others with the same issue here is some additional detail on how I solved it:
The prerelease did work, HOWEVER since I had already installed the `inky` package and some pre-requisites with `sudo`  (as per instructions/guides) which needed to be rectified.

1. Install pre-release per #182 
2. Attempt to run examples... doesn't work because it can't find smbus2
3. Deactivate virtual env and uninstall smbus and smbus2 as root from both python2/3 :
```
sudo pip uninstall smbus2 / sudo pip uninstall smbus
sudo pip3 uninstall smbus2 / sudo pip3 uninstall smbus
```
4. Reactivate environment then install smbus (not as root) `pip install smbus2` 
5. Now the examples work (mostly)

For all other packages (such as the fonts, numpy etc)  that were installed as root, repeat steps 3 & 4 to fix.
